ui/app/adapters/pki/issuer.js

```diff
@@ -31,6 +33,35 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  async getIssuerMetadata(store, type, query, response) {
+    const { backend } = query;
+
+    const keyInfoArray = await all(
+      response.data.keys.map((id) => {
```

Comment:
This is sort of a personal nit, but I don't love mixing then/catch and async/await style promises. I think this could be cleaner if we pulled the method into an internal helper method which is async/await style. Not a blocker, just would make it a tad more legible IMO!

```suggestion
      response.data.keys.map((id) => this.queryRecordWithInfo(store, type, query, response, id))
```
then...
```
async queryRecordWithInfo(store, type, query, response, id) {
  const keyInfo = response.data.key_info[id];
  try {
    const issuerRecord = await this.queryRecord(...)
    // ... more business logic
  } catch (e) {
    return {...keyInfo}
  }
}
```